README.md

```diff
@@ -1,152 +1,34 @@
 ```bash
-docker-compose down
-```
-
-## **Without Docker Compose**
-Though this project was built for easy startup with Docker, it can also be run without Docker. You would need to run the commands on two terminals, one for the *frontend* and the other for the *backend*:
-
-## **Frontend**
-On one terminal, go to the frontend folder named `client`, install dependencies and run the development server:
-```bash
-cd client
-npm install
 npm run dev
+# or
+yarn dev
 ```
 
+Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
 
-## **Backend**
-On another terminal, create and activate a virtual environment, and then install the requirements.
+You can start editing the page by modifying `pages/index.tsx`. The page auto-updates as you edit the file.
 
-### Set Environment Variables
+[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.ts`.
 
-Update *project/server/config.py*, and then run:
+The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
 
-```sh
-$ cd backend
-$ export APP_NAME="ALIAS CHECK API"
-$ export APP_SETTINGS="app.config.ProductionConfig"
-$ export FLASK_DEBUG=0
-```
-By default the app is set to use the production configuration. If you would like to use the development configuration, you can alter the `APP_SETTINGS` environment variable:
+## Learn More
 
-```sh
-$ export APP_SETTINGS="app.config.DevelopmentConfig"
-```
-
-Using [Pipenv](https://docs.pipenv.org/) Use the *.env* file to set environment variables:
-
-```
-APP_NAME="ALIAS CHECK API"
-APP_SETTINGS="app.config.DevelopmentConfig"
-FLASK_DEBUG=1
-```
-
-### Create DB
+To learn more about Next.js, take a look at the following resources:
 
-```sh
-$ python manage.py create-db
-$ python manage.py db init
-$ python manage.py db migrate
-```
-
-### Run the Application
-
-
-```sh
-$ python manage.py run
-```
+- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
+- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
 
-Access the application at the address [http://localhost:5000/](http://localhost:5000/)
+You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
 
-### Testing
+## Deploy on Vercel
 
-Without coverage:
+The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
 
-```sh
-$ python manage.py test
-```
-
-With coverage:
-
-```sh
-$ python manage.py cov
-```
-
-Run flake8 on the app:
-
-```sh
-$ python manage.py flake
-```
-
-or
-
-```sh
-$ flake8 project
-```
```

Comment:
I did not make any change to the README. The README will be updated as part of issue https://github.com/KanaryStack/aliascheck/issues/28

Reply:
Here's [the change your PR makes to the README](https://github.com/KanaryStack/aliascheck/pull/26/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L1-L152). This change occurs because many of the changes you make were done in the root directory and I think the change to the README was unintended.